ciclo/callbacks.py

```diff
@@ -55,6 +55,11 @@ class OptimizationMode(str, Enum):
     max = auto()
 
 
+class DeltaMode(str, Enum):
```

Comment:
I used the same convention as the existing `mode` argument (now called `optimization_mode`) , i.e., using an `Enum` to represent the options. For consistency, perhaps it makes sense to define a similar `Enum` for the `InnerLoopAggregation` options introduced in #9?  (Or we could use that PR's approach of defining a string `Literal` type for the different options?)